### DISCUSSION TITLE
How to configure so only prometheus is installed without alertmanager nor grafana

### DISCUSSION THREAD

**Discussion by awangc** [prometheus-operator/kube-prometheus#636] [General] [answered]
I'm only interested in collecting metrics inside the kubernetes cluster, and exporting metrics for an external prometheus (not in the k8s cluster) that will perform alerting.

How do I configure kube-prometheus so only prometheus (plus service monitors, node exporters, etc - but no alertmanager nor grafana) is installed?

Is there anything else that can acts as a metrics collector other than prometheus inside a k8s cluster?

**Comment by brancz** [comment, marked as answer]
You can take the example from [here](https://github.com/prometheus-operator/kube-prometheus#compiling), and just comment out the parts you are not interested at the end, that way those components' manifests won't be rendered.

**Reply by awangc** [reply]
Thanks!
